ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/FontAwesome.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.ide.ui;
+package org.eclipse.che.ide.api;
```

Comment:
Why have you moved _FontAwesome_ class from the **che-core-ide-ui** to the **che-core-ide-api** module?